emepuchades/task-manager
# Add eslint and prettier configuration

# Task Manager Pull Request 📅

Add eslint and prettier configuration to format and lint the code

## Descripcion ✏️
Every time the dev adds new code, he/she has to run npm run format and npm run lint to fix any format issues that may appear. I strongly suggest that the IDE config has the option "Format on save action" enabled

## Visual Changes 🎨
My PR is a configuration preset, so no visual changes to validate

## How to test 🐛
Run npm run format and after that, npm run lint and make sure any format issues that may occur are addressed correctly
